src/battle_anim_effects_1.c

```diff
@@ -4452,7 +4452,14 @@ static void AnimBowMon_Step1_Callback(struct Sprite *sprite)
     {
         sprite->data[3] = gBattlerSpriteIds[gBattleAnimAttacker];
         PrepareBattlerSpriteForRotScale(sprite->data[3], ST_OAM_OBJ_NORMAL);
+#ifdef CLANGD
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wconstant-conversion"
+#endif
         sprite->data[4] = (sprite->data[6] = GetBattlerSide(gBattleAnimAttacker)) ? 0x300 : 0xFFFFFD00;
+#ifdef CLANGD
+#pragma clang diagnostic pop
+#endif
```

Comment:
Is there some other way this could be written, or is the problem that `0xFFFFFD00` needs to be written literally like that and can't be `-0x300` (or failing that, `0xFD00`)?

It's a shame to turn one line into eight :sweat_smile: